app/src/main/java/io/stanwood/bitrise/data/model/RepoProvider.kt

```diff
@@ -0,0 +1,8 @@
+package io.stanwood.bitrise.data.model
+
+import com.google.gson.annotations.SerializedName
+
+enum class RepoProvider {
+    @SerializedName("github")
```

Comment:
So there is only github? What about gitlab etc?

Reply:
1. I don't know all the providers supported by Bitrise
2. Will add more providers later

Reply:
Will it crash when something else arrives here?